From fa2d0881ff3084977aa01dc9a4254a26d70245c5 Mon Sep 17 00:00:00 2001
From: Vinitha V Pillai <vinitha.pillai@nxp.com>
Date: Wed, 12 Sep 2018 15:34:30 +0530
Subject: [PATCH 104/107] ls1088ardb_pb: Secure boot: Change bootscript and header name

Change bootscript and its header name in
ls1088ardb_pb board specific file

Signed-off-by: Vinitha V Pillai <vinitha.pillai@nxp.com>
---
 include/configs/ls1088ardb_pb.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/ls1088ardb_pb.h b/include/configs/ls1088ardb_pb.h
index 8f4612a..49ec737 100644
--- a/include/configs/ls1088ardb_pb.h
+++ b/include/configs/ls1088ardb_pb.h
@@ -63,8 +63,8 @@
 	"kernelhdr_size_sd=0x10\0"		\
 	MC_INIT_CMD				\
 	BOOTENV					\
-	"boot_scripts=ls1088ardb_boot.scr\0"	\
-	"boot_script_hdr=hdr_ls1088ardb_bs.out\0"	\
+	"boot_scripts=ls1088ardb_pb_boot.scr\0"	\
+	"boot_script_hdr=hdr_ls1088ardb_pb_bs.out\0"	\
 	"scan_dev_for_boot_part="		\
 		"part list ${devtype} ${devnum} devplist; "	\
 		"env exists devplist || setenv devplist 1; "	\
-- 
1.7.1

